routing/routing.xml

```diff
@@ -549,7 +561,15 @@
 			<select value="-1" t="mtb:scale" v="5"/>
 			<select value="-1" t="mtb:scale" v="6"/>
 
-			<select value="-1" t="vehicle" v="no"/>
+			<select value="1" t="vehicle" v="no"/>
+			<!-- The tag vehicle=no is normally used when mappers
+			want to exclude motor vehicles. I have never seen a case
+			when such a road was actually forbidden for bicycles as
+			well. (Mappers often do not bother to add the bicycle=yes).
+			Example:www.openstreetmap.org/way/159679495  This is
```

Comment:
This is wrong tagging. According to wiki vehicle=no forbids way for all vehicles (motorized and non-motorized). Also, access=no forbids access to every person (vehicle, pedestrian etc). So I disagree. Proper tagging on this way should be motor_vehicle=no if I understand mapper's intentions correctly.